pkg/api/dirs.go

```diff
@@ -63,6 +64,16 @@ func (s *Service) dirUploadHandler(
 	}
 	defer r.Body.Close()
 
+	rLevelNum, err := strconv.ParseUint(r.Header.Get(SwarmRedunancyLevel), 10, 1)
```

Comment:
What is the ,1?   According to the docs:

```The bitSize argument specifies the integer type that the result must fit into. Bit sizes 0, 8, 16, 32, and 64 correspond to int, int8, int16, int32, and int64. If bitSize is below 0 or above 64, an error is returned.```

Reply:
great catch ;)